### PR TITLE
Update info about NumPy newcomers meetings

### PR DESCRIPTION
**Pull request by InessaPawson**

### CODE REVIEW

File: calendars/numpy.yaml [hunk 1]
```diff
@@ -44,13 +44,30 @@ events:
...
+    begin: 2022-04-21 16:00:00 +00:00
+    end: 2022-04-21 17:00:00 +00:00
     repeat:
       interval:
         # seconds, minutes, hours, days, weeks, months, years
         days: 14
       until: 2025-01-01 00:00:00 +00:00
+      
+      - summary: NumPy Newcomers meeting
+    description: |
```
Comment (tupui) [inline calendars/numpy.yaml:62]: Not sure about the title, and there is an indentation issue
```suggestion
  - summary: Making NumPy accessible with community workshops
    description: |
```

File: calendars/numpy.yaml [hunk 2]
```diff
@@ -44,13 +44,30 @@ events:
...
-      The fortnightly NumPy call for newcomers.
+      This is a meeting with no agenda, meant to provide newcomers with the space 
+      to ask questions, meet other contributors, and chat about NumPy. 
+      Drop in any time and stay as long as you wish.
+      <br />
+      Join us via Zoom: https://us02web.zoom.us/j/87192457898
 
-    begin: 2022-02-10 16:00:00 +00:00
-    end: 2022-02-10 17:00:00 +00:00
-    url: https://us02web.zoom.us/j/87192457898
```
Comment (tupui) [inline calendars/numpy.yaml:51]: I think we want to keep this for calendars to present the url correctly no?

**Review by tupui — COMMENTED** [review]
Thank you @InessaPawson 👍 I have a few suggestions

**Comment by jarrodmillman** [comment]
@InessaPawson @tupui Should we fix this PR and merge it? If not, should we close it? Thanks!

**Comment by InessaPawson** [comment]
> @InessaPawson @tupui Should we fix this PR and merge it? If not, should we close it? Thanks!

Please close it.
I’ve been taking notes on the needs for our community calendar. I’ll submit a new PR that incorporates my findings.

**Pull request closed without merging**